holiday-calendar-eur/src/main/java/com/github/davejoyce/calendar/impl/HolidayCalendarServiceCH.java

```diff
@@ -0,0 +1,43 @@
+package com.github.davejoyce.calendar.impl;
+
+import com.github.davejoyce.calendar.Holiday;
+import com.github.davejoyce.calendar.HolidayCalendar;
+import com.github.davejoyce.calendar.HolidayCalendarService;
+
+import java.time.Month;
+
+import static com.github.davejoyce.calendar.HolidayCalendar.STANDARD_WEEKEND;
+
+/**
+ * Service for provision of Switzerland national holiday calendar.
+ *
+ * @author <a href="mailto:dave@osframework.org">Dave Joyce</a>
+ */
+public class HolidayCalendarServiceCH implements HolidayCalendarService {
+
+    private static final String CODE = "CH";
+    private static final String NAME = "Switzerland National Holidays";
```

Comment:
I'm not sure where this string will show, up it feels grammatically clunky.  If this were in an English sentence I would expect it to appear as "Swiss National Holidays" or "National Holidays of Switzerland".  But, it does need to be consistent with the other countries (although Canada arguably has the same clunkiness), so unless we're changing all the others this should probably stay as is for now.